Overview.src.html

```diff
@@ -202,6 +202,34 @@ <h2 class="short">Conformance</h2>
 <span>process request body</span> for <var>request</var>.
 <!-- XXX xref "read", "payload body" -->
 
+<h3>Token Binding</h3>
+
+<p>The technique of Token Binding allows user agents to work with servers to protect credentials
```

Comment:
I refactored everything, so that the bulk of the changes are now in the "Token Binding" section.

Please let me know if this is what you had in mind.